label_studio/ml_model_providers/models.py

```diff
@@ -27,6 +27,8 @@ class ModelProviderConnection(models.Model):
 
     endpoint = models.CharField(max_length=512, null=True, blank=True, help_text='Azure OpenAI endpoint')
 
+    available_models = models.CharField(max_length=4096, null=True, blank=True, help_text='List of available models from the provider')
```

Comment:
nit: wdyt of something like "cached_available_models"? to make it clearer how/why we're using it

Reply:
Yep will change it